docs/best-practices/xcodebuild-formatters.md

```diff
@@ -55,16 +55,16 @@ $ xcodebuild [flags] | <xcodebuild_formatter>
 $ brew install xcbeautify
 ```
 
-`xcbeautify` is the recommended because:
- 
- - Faster than `xcpretty`
+`xcbeautify` is the recommended one because it:
+
  - Supports Xcode's new build system output
- - Supports  Xcode's parallel testing output
+ - Supports Xcode's parallel testing output
  - Supports formatting Swift Package Manager output
+ - It's faster than `xcpretty`
```

Comment:
I debated this when I wrote it haha I opted to use `It's` because when read in isolation it sounds a lot better than `is` (although I acknowledge that `is``would be "more correct" here 😄 )

Reply:
As a non-English speaker I don't usually have too much of an insight into such fine details in wording, so I'd like to leave that to someone better equipped for such assessment ;D

It's just an observation, if the heading starts with "because it: …" I'm sort of expecting the lines to be an ending to this sentence. That's why my gut feeling kinda expects every line to actually a) start with lowercase, b) finish the sentence. Like:
```suggestion
 - is faster than…
 - supports Xcode…
 - supports formatting…
```

(but that stylisation is not present in other places of the text with all bullet lines being capitalised even when they form a part of a sentence… hence my hesitation to suggest a "correction" as I'm not too skilled in language matters TBH;D…)

Reply:
That makes sense but makes skimming through the page a bit more confusing and requires more reading (cognitive friction) IMO. I've updated it to something else that might help, lmk what you think @janbrasna 😊

Reply:
I've checked the copy style in other parts of the docs and yes, they are all sentence-case (w/o a closing full stop) when used as bullet/list items usually used in an infinitive form following the preceding intro text. So please disregard any comments I may have made.

The only form I could've come up with is to reword in a way to make use of the infinitives like:

> … is currently recommended as it was designed to:
>  - Support Xcode…
>  - Support formatting…
>  - Be faster than…

(or any cumbersome formulations as "it is known to:", "it is intended to:" but that's all weird compared to just saying it's what it is…) — besides that I'm out of any drive-by improvements, sorry;)